### PR TITLE
Windows driver checkbox

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.13

Issue: None

Link to docs preview: [Disks tab](https://58110--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by apinnick** [comment]
@gouyang Could you review this PR? It's very small. Thanks.

File: virt/virt-web-console-overview.adoc [hunk 1]
```diff
@@ -771,6 +771,9 @@ You can manage disks on the *Disks* tab.
 |Search field
 |Search for a disk by name.
 
+|*Mount Windows driver disk* checkbox
+|Mounts an ephemeral container disk as a CD-ROM
```
Comment (kcarmichael08) [inline virt/virt-web-console-overview.adoc:775]: I would add a period since it's a sentence (it looks like the other list items follow that convention).

**Review by kcarmichael08 — COMMENTED** [review]
One tiny suggestion; looks good to me!

File: virt/virt-web-console-overview.adoc (hunk 1 above)
Comment (apinnick) [reply virt/virt-web-console-overview.adoc:775]: I think it's a sentence fragment. However, I can rewrite this as "Select to mount ...etc."

**Pull request merged**